style.css

```diff
@@ -277,21 +305,58 @@ body {
   flex-direction: column;
   gap: 2.4rem;
   padding-top: 2.4rem;
-  padding-bottom: 12rem;
+  padding-bottom: 2rem;
 }
 
-.feature--1,
-.feature--2,
-.feature--3,
-.feature--4,
-.feature--5,
-.feature--6 {
+.open-modal {
+  width: 90%;
+  height: 90vh;
+  background-color: #fff;
+  position: fixed;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  padding: 1.6rem;
+  padding-bottom: 2.4rem;
+  margin: auto;
+  z-index: 5;
+  box-shadow: 0 0 20px rgb(84, 84, 84);
+}
```

Comment:
- [ ] As the previous reviewer suggested, Kindly try to match your popup design to the Figma design. You can reduce the space between your image and the bottom of the popup.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![pop-figma](https://github.com/Alibaba2023/My_Portfolio/assets/70483835/4b7ce25b-f719-4b64-8e1f-3add9311a000) | ![pop_fig](https://github.com/Alibaba2023/My_Portfolio/assets/70483835/cda0c3f8-8a72-4e89-b507-c2ec747958b2)

Reply:
Hi  @mahbubul14  ;
## 
![Screenshot 2023-05-19 071342](https://github.com/Alibaba2023/My_Portfolio/assets/126192888/72ef3b9c-bd7e-4bda-8e8f-6564866cfa1e)
But the popup menu on my computer is like this:-

Reply:
@mahbubul14 
Can you check the project again?
On my computer, the popup is working well.